### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFBrush_Contents_ScreenReader: Using narrator the expand/collapse state not announced for the "Contents" button for screen reader users.der user.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1456]
Copied from VSO [948255](https://devdiv.visualstudio.com/DevDiv/_queries/edit/948255)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF and ASP.NET Core)

Application version: VS 2019 Preview 4.7.03190

Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link.

Repro Steps:

1) Open Visual Studio.
2) Hit on File and select Open->Project/Solution.
3) Select the "Brushes" solution and open it. 
4) From the Solution Explorer, right click on the file under the solution "Brushes".
5) Select "Debug" and hit "Start new Instance".
6) Main Window for "Brush Examples" solution will appear.
7) Start screen reader.
8) Navigate to "Contents" section and listen if the expand/collapse state is announced properly.

Actual:

Using narrator when screen reader users navigate to the "Contents" section first "Contents grouping collapsed" is announced and then the "Contents" button is announced. The expand/collapse action takes place on activating the "Content" button but the state here is not announced to the screen reader users.

Same issue is observed in the below scenario:

1) Linear Gradient - Interactive Example - The expand/collapse action takes place on activating the "Markup" button but the state here is not announced to the screen reader users.

Expected:

The proper expand/collapse state should be announced on the "Content" button so that when screen reader user activates the button element it should inform then that whether some content has expanded or collapsed. 

User Impact:

If proper state of the interactive element is not announced to the screen reader users then will not understand on activating the element some action has taken place for not. 

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by arpitmathur** [comment]
When I navigate to the Contents group using Narrator, I hear "Contents group collapsed". I am able to activate it by pressing Enter and Narrator announces "Contents group expanded" as expected.
